modules/statics/src/coins/erc20Coins.ts

```diff
@@ -0,0 +1,44 @@
+import { UnderlyingAsset } from '../base';
+import { erc20 } from '../account';
+export const erc20Coins = [
+  erc20(
+    '3cdd1d41-b561-4c0c-aa82-72c52ebe69e5',
+    'eth:kava',
+    'Kava.io',
+    6,
+    '0x0c356b7fd36a5357e5a017ef11887ba100c9ab76',
+    UnderlyingAsset['eth:kava']
+  ),
+  erc20(
+    '0b22028a-fa33-47e3-9b4d-b32173b53ab6',
+    'eth:iq',
+    'Everipedia IQ',
+    8,
```

Comment:
https://etherscan.io/token/0x579cea1889991f68acc35ff5c3dd0621ff29b0c9
It should be 18 decimals as per link

Reply:
Good catch. Thank you for noticing this. 🙇